api_fhir_r4/views/fhir/practitioner.py

```diff
@@ -22,8 +27,10 @@ class PractitionerViewSet(BaseMultiserializerFHIRView,
     @property
     def serializers(self):
         return {
-            ClaimAdminPractitionerSerializer: (self._ca_queryset(), self._ca_serializer_validator),
-            EnrolmentOfficerPractitionerSerializer: (self._eo_queryset(), self._eo_serializer_validator),
+            ClaimAdminPractitionerSerializer:
+                (self._ca_queryset(), self._ca_serializer_validator, (FHIRApiPractitionerPermissions,)),
+            EnrolmentOfficerPractitionerSerializer:
+                (self._eo_queryset(), self._eo_serializer_validator, (FHIRApiPractitionerPermissions,)),
```

Comment:
FHIRApiPractitionerPermissionsis hard-coded to checkclaim admin permissions. Can you split it for relevant resources?